containerd-shim-spin-config.md

```diff
@@ -0,0 +1,372 @@
+# SKIP 00X - Configuring Runtime Options in the `containerd-shim-spin`
+
+Summary: Configuring Spin runtime options in the `containerd-shim-spin`
+
+Owner: Kate Goldenring <kate.goldenring@fermyon.com> 
+
+Impacted Projects:
+
+- [ ] spin-operator
+- [ ] `spin kube` plugin
+- [x] runtime-class-manager
+- [x] containerd-shim-spin
+- [ ] Governance
+- [ ] Creates a new project
+
+Created: 30/05/2024
+
+## Overview
+
+There is currently no defined way to configure the execution of the
+`containerd-shim-spin` SpinKube executor. This proposal aims to address this at
+two levels: Spin application execution configuration and Shim execution
```

Comment:
```suggestion
two levels: Spin application execution configuration (Spin runtime and components) and Shim execution (containerd shim)
```

Something like that?  I didn't know what these meant until I got further along in the document.